components/README.md

```diff
@@ -2,45 +2,57 @@
 
 SUI is an open-source, high-quality library of reusable React components that empower teams to craft any product with ease.
 
-## Declarative approach:
+## Declarative approach
 
 SUI is Declarative, Intuitive, Consistent and Product-agnostic. Every component should allow its customisation through React Props.
 
 If you need to add or adjust any of the following for a component, you can send a PR directly without previous notice.
 
-- Font sizes, colour and weight
-- Border visibility, size and colour
-- Border radius
-- Box-Shadow style
-- Background colour
-- Paddings and Margins
-- Icon and icon colour
+<ul>
+<li>Font sizes, colour and weight</li>
+<li>Border visibility, size and colour</li>
+<li>Border radius</li>
+<li>Box-Shadow style</li>
+<li>Background colour</li>
+<li>Paddings and Margins</li>
+<li>Icon and icon colour</li>
```

Comment:
`Icon` and `icon` ?

Reply:
fixed, thanks
I replaced "icon" for "its"

Reply:
Nice! 👍